observation_portal/accounts/urls.py

```diff
@@ -14,6 +15,7 @@
     re_path(r'^register/$', CustomRegistrationView.as_view(form_class=CustomRegistrationForm), name='registration_register'),
     re_path(r'^loggedinstate/$', TemplateView.as_view(template_name='auth/logged_in_state.html'), name='logged-in-state'),
     path("login/", CustomLoginView.as_view(), name="auth_login"),
+    path("api-login/", ApiLoginView.as_view(), name="api_login"),
```

Comment:
Maybe it would make sense to put this under the `/api/` path, so its `/api/login/`?

Reply:
yea I dunno that would require importing from the accounts.views module in some different places since the root urls.py does an include, kinda nice that all the account stuff is contained here in my opinion.